scipy/stats/_stats_py.py

```diff
@@ -9770,16 +9784,81 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
 
     where :math:`\Gamma (u, v)` is the set of (probability) distributions on
     :math:`\mathbb{R} \times \mathbb{R}` whose marginals are :math:`u` and
-    :math:`v` on the first and second factors respectively.
+    :math:`v` on the first and second factors respectively. For a given value
+    :math:`x`, :math:`u(x)` gives the probabilty of :math:`u` at position
+    :math:`x`, and the same for :math:`v(x)`.
 
-    If :math:`U` and :math:`V` are the respective CDFs of :math:`u` and
-    :math:`v`, this distance also equals to:
+    In the 1-dimensional case, let :math:`U` and :math:`V` denote the
+    respective CDFs of :math:`u` and :math:`v`, this distance also equals to:
 
     .. math::
 
         l_1(u, v) = \int_{-\infty}^{+\infty} |U-V|
 
-    See [2]_ for a proof of the equivalence of both definitions.
+    See [3]_ for a proof of the equivalence of both definitions.
+
+    In the more general (higher dimensional) and discrete case, it is also
+    called the optimal transport problem or the Monge problem.
+    Let the finite point sets :math:`\{x_i\}` and :math:`\{y_j\}` denote
+    the support set of probability mass function :math:`u` and :math:`u`
+    respectively. The Monge problem can be expressed in this form,
+
+    Let :math:`\Gamma` denote the transport plan, :math:`D` denote the
+    distance matrix and follows,
+
+    .. math::
+
+        x = \text{vec}(\Gamma)          \\
+        c = \text{vec}(D)               \\
+        b = \begin{bmatrix}
+                u\\
+                v\\
+            \end{bmatrix}
+    The :math:`\text{vec}()` function denotes the Vectorization function 
```

Comment:
```suggestion

    The :math:`\text{vec}()` function denotes the Vectorization function 
```

Reply:
Thanks, I will have a look at this later today. Also, is it possible I could test the doc locally like rendering only the specific page?

Reply:
Try `python dev.py doc` to build docs locally. If you have trouble, follow the latest build instructions in this PR: https://github.com/scipy/scipy/pull/18429